crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -203,18 +215,32 @@ def provided_with_enabled?(feature, platform = nil, arch = nil, check_enabled =
             found = true
 
             r = new(platform, arch, repo)
+
             answer &&= r.available?
+            unless answer
```

Comment:
ah, okay. now I know what the operator is doing... I just assumed `answer = r.available?` here... my bad